documentation/running_bolt_commands.md

```diff
@@ -1,418 +1,604 @@
+
+## Specify a transport
+
+Bolt uses a specific transport to establish a connection with a target. By
+default, Bolt connects to targets using the `ssh` transport. You can use one of
+the methods below to set a different transport from the command line, or you can
+configure transports in your inventory file.
+
+You can specify the transport used to connect to a specific target by setting
+it as the protocol in the target's URI:
+
+_\*nix shell command_
+
+```shell
+bolt command run 'Get-Location' --targets winrm://windows.example.org
 ```
 
-If `localhost` is passed to `--targets` when invoking Bolt,
-the `local` transport is used automatically. To avoid this behavior, prepend the
-target with the desired transport, for example `ssh://localhost`.
+_PowerShell cmdlet_
 
+```powershell
+Invoke-BoltCommand -Command 'Get-Location' -Targets winrm://windows.example.org
+```
 
-### Specify connection credentials
+You can also use the `transport` command-line option:
 
-To manage a target with Bolt, you must specify credentials for a user on the
-target. You have several options for doing this, depending on which operating
-system the target is running.
+_\*nix shell command_
 
-Whether the target runs Linux or Windows, the simplest way to specify
-credentials is to pass the username and password right in the Bolt command:
+```shell
+bolt command run 'Get-Location' --targets windows.example.org --transport winrm
 ```
-bolt command run 'hostname' --targets <LINUX_TARGETS> --user <USER> --password <PASSWORD>
+
+_PowerShell cmdlet_
+
+```powershell
+Invoke-BoltCommand -Command 'Get-Location' -Targets windows.example.org -Transport winrm
 ```
 
-If you'd prefer to have Bolt securely prompt for a password (so that it won't
-appear in a process listing or on the console), use the `--password-prompt`
-option without including a value:
+📖 **Related information**
+
+- [Bolt transports reference](bolt_transports_reference.md)
+
+## Run a script
+
+When you run a script on a remote target, Bolt copies the script from your
+workstation to a temporary directory on the target, runs the script, and then
+deletes the script from the target.
+
+You can run scripts in any language, as long as the appropriate interpreter is
+installed on the remote system. This includes scripting languages such as Bash,
+PowerShell, Python, and Ruby.
+
+To run a script, provide the path to the script on the workstation and a list of
+targets to run the script on.
+
+_\*nix shell command_
+
+```shell
+bolt script run ./scripts/configure.sh --targets servers
 ```
-bolt command run 'hostname' --targets <LINUX_TARGETS> --user <USER> --password-prompt
+
+_PowerShell cmdlet_
+
+```powershell
+Invoke-BoltScript -Script ./scripts/configure.ps1 -Targets servers
 ```
 
-If the target runs Linux, you can use a username and a public/private key pair
-instead of a password:
+### Pass arguments to a script
+You can also pass arguments to a script. Argument values are passed literally
```

Comment:
```suggestion
Argument values are passed literally
```
This is a passive, but I'm not sure how to make it active. "The shell on the remote host does not interpolate script values, Bolt passes the argument values literally." Perhaps we just leave this one in passive voice